packages/dbos-cloud/login.ts

```diff
@@ -1,6 +1,6 @@
-import { GlobalLogger } from "../telemetry/logs";
+import { GlobalLogger } from "../../src/telemetry/logs";
```

Comment:
would it be possible (and also desirable) to tweak the `tsconfig.json` `path` property to allow imports of the like `import {...} from telemetry/logs` ?

Reply:
I found it not desirable right now.